Redesigned UI

jonathantjendana opened the pull request:
![image](https://user-images.githubusercontent.com/22949830/67799710-da9e8f00-fac0-11e9-9513-93b2017c7b99.png)
![image](https://user-images.githubusercontent.com/22949830/67799784-fdc93e80-fac0-11e9-9dae-340f23200d0c.png)

jonathantjendana committed:
Redesigned UI

jonathantjendana committed:
Merge branch 'master' into branch-ui

jonathantjendana committed:
Removed unused import

choongyx commented on src/main/java/seedu/moneygowhere/ui/GraphPanel.java:
```diff
@@ -0,0 +1,62 @@
+ */
+public class GraphPanel extends UiPart<Region> {
+    private static final String FXML = "PlaceholderPanel.fxml";
+    private final Logger logger = LogsCenter.getLogger(GraphPanel.class);
```
Can remove the loggers if unused in order to pass Codacy test.

jonathantjendana replied:
Done!

choongyx reviewed (approved):
Looks good! 👍👍👍

Can try deleting some of the unused files e.g statsChart and graphChart fxml and java files to increase coverage.

minpyaemoe commented:
![image](https://user-images.githubusercontent.com/13093826/67824366-16594900-fb01-11e9-81f7-56f78883ad63.png)

The black color pane is intended, right?

minpyaemoe reviewed (commented):
![image](https://user-images.githubusercontent.com/13093826/67824436-502a4f80-fb01-11e9-8d19-f18eace96ef2.png)

The black color extra pane under the items are intended, right?

Nanosync commented:
This is a good step towards a final design! I like the separators, very creative. However, I hope that the balance bar can be changed later. The reminders also seem to take up a lot of space and can be tweaked.

I cloned the repository and found some bugs:
- The background of the list panel is incorrect
![image](https://user-images.githubusercontent.com/10258208/67826554-d77ac180-fb07-11e9-9adf-58ae4ff0d9d3.png)

- The graph/stats panel items keep shifting when you click another tab pane and hit any key or focus on another element.
![image](https://user-images.githubusercontent.com/10258208/67826589-fb3e0780-fb07-11e9-8e57-608652b462fa.png)
![image](https://user-images.githubusercontent.com/10258208/67826709-6d165100-fb08-11e9-99a4-5852e43a680f.png)

jonathantjendana committed:
- Removed unused loggers
- Removed unused files
- Changed listview background colour
- Bugfix moving charts

jonathantjendana committed:
Removed unused imports
